sw/airborne/modules/nav/nav_geofence.h

```diff
@@ -30,8 +30,7 @@
  * home_mode_max_alt is (optionally) defined in the flight plan
  * GEOFENCE_DATALINK_LOST_TIME is defined in the airframe config file
  */
-
-#include <stdbool.h>
+#include "subsystems/datalink/datalink.h"
```

Comment:
I would place this line inside the `#ifdef GEOFENCE_DATALINK_LOST_TIME` so that it is only included if really needed